### DISCUSSION TITLE
Soft-deprecated properties

### DISCUSSION THREAD

**Discussion by pan-zhuo** [uwlib-cams/sinopia_maps#86] [decided]
> An element that is soft-deprecated is one that is redundant with a recording method of another element or that is considered to be an unnecessary refinement of another element.

From RDA Registry [Alignment from soft-deprecated to recommended elements](http://www.rdaregistry.info/Aligns/alignSoft2Rec.html)

In the Toolkit, the Prerecording instruction of such an element will point to the recommended element.

![image](https://user-images.githubusercontent.com/100716525/221460538-5aa83a60-2ae0-450e-be73-a4f507292297.png)

## Problem

Some of these elements are LC-PCC Core, such as Parallel title proper.

Are we going to
  - keep in line with PCC,
  - not use any of them, or
  - decide case-by-case?

## See also

Related discussion in MARC2RDA [#374](https://github.com/uwlib-cams/MARC2RDA/discussions/374)

**Comment by briesenberg07** [comment]
My opinions below:

1. **For this specific case, use [parallel title proper](http://rdaregistry.info/Elements/m/P30203) (deprecated) or repeat [title proper](http://rdaregistry.info/Elements/m/P30156)?**

      *Repeat **title proper**. Where a title proper in a preferred language is needed (or value for parallel title proper is needed for some reason), language tags can be used to retrieve the needed value. (But this does depend on the availability of accurate language tags.)*

2. **In general, use a LC-PCC Core element which is deprecated, or the element recommended to replace it?**

      *Evaluate case-by-case with a preference for using the recommended replacement.*

These are my answers because I believe our aim is to model and create well-formed RDA/RDF which aligns with best practices for implementation of the RDA ontology.

To my mind, because BIBFRAME is LC/PCC's preferred model for RDF library data, alignment with LC/PCC practices is not a primary consideration when modeling RDA/RDF. 

Conversion of RDA/RDF data to BIBFRAME might be a future goal. In this scenario LC/PCC best practices would be a primary consideration, and might influence the selection of elements for an RDA/RDF MAP. (In the specific case above, however, it seems to me that we would still be able to retrieve the data needed for the LC-PCC core element from RDA/RDF modeled according to best practices.)

**Comment by briesenberg07** [comment]
**📢 DECISION 2023-03-03:**

Don't use soft-deprecated elements.
